src/core/worker.js

```diff
@@ -515,6 +515,18 @@ class WorkerMessageHandler {
       });
     });
 
+    handler.on("GetAnnotationObjects", function ({ numPages }) {
+      const promises = [];
+      for (let pageIndex = 0; pageIndex < numPages; pageIndex++) {
+        promises.push(
+          pdfManager.getPage(pageIndex).then(function (page) {
+            return page.getAnnotationObjects();
+          })
+        );
+      }
+      return Promise.all(promises).then(data => data.flat());
```

Comment:
Given that there's not been any recent decisions, as far as I'm aware, of the lowest supported Chrome/Safari versions you'll need to add a polyfill for this Array method in `src/shared/compatibility.js`; similar to e.g. https://github.com/mozilla/pdf.js/blob/88f72d6b1c37dfc0dbb6715685bf8cb4ae516cd7/src/shared/compatibility.js#L81-L97

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat#Browser_compatibility